### PR TITLE
Remove references to thoas

### PR DESCRIPTION
**Pull request by azangru**
## Description
- replaced the url that had thoas in its name (`/api/thoas` becomes `/api/graphql/core`)
- renamed the directory containing types of entities retrieved from Thoas from `thoas` to `core-api`
- updated several remaining variable names

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1977

## Deployment URL(s)
http://replace-thoas-url.review.ensembl.org

### CODE REVIEW

**Review by kamaldodiya — APPROVED** [review]
Have update k8s manifest and added CORE_API_BASE_URL. Also keeping THOAS_BASE_URL which needs to be removed at some point

File: src/content/app/entity-viewer/gene-view/components/gene-overview-image/GeneOverviewImage.tsx [hunk 1]
```diff
@@ -30,8 +30,8 @@ import UnsplicedTranscript, {
 } from 'src/content/app/entity-viewer/gene-view/components/unspliced-transcript/UnsplicedTranscript';
 import FeatureLengthRuler from 'src/shared/components/feature-length-ruler/FeatureLengthRuler';
 
-import { FullGene } from 'src/shared/types/thoas/gene';
-import { FullTranscript } from 'src/shared/types/thoas/transcript';
+import { FullGene } from 'src/shared/types/core-api/gene';
+import { FullTranscript } from 'src/shared/types/core-api/transcript';
 import { TicksAndScale } from 'src/shared/components/feature-length-ruler/FeatureLengthRuler';
```
Comment (imransl) [inline src/content/app/entity-viewer/gene-view/components/gene-overview-image/GeneOverviewImage.tsx:35]: Change these to type imports?

File: src/content/app/entity-viewer/gene-view/components/unspliced-transcript/UnsplicedTranscript.tsx [hunk 2]
```diff
@@ -19,9 +19,9 @@ import classNames from 'classnames';
 import { scaleLinear, ScaleLinear } from 'd3';
 import { Pick2, Pick3 } from 'ts-multipick';
 
-import { FullTranscript } from 'src/shared/types/thoas/transcript';
-import { SplicedExon } from 'src/shared/types/thoas/exon';
-import { FullCDS } from 'src/shared/types/thoas/cds';
+import { FullTranscript } from 'src/shared/types/core-api/transcript';
+import { SplicedExon } from 'src/shared/types/core-api/exon';
+import { FullCDS } from 'src/shared/types/core-api/cds';
```
Comment (imransl) [inline src/content/app/entity-viewer/gene-view/components/unspliced-transcript/UnsplicedTranscript.tsx:24]: Same as above (type imports)

File: src/content/app/entity-viewer/gene-view/components/transcripts-filter/TranscriptsFilter.tsx [hunk 3]
```diff
@@ -42,7 +42,7 @@ import Checkbox from 'src/shared/components/checkbox/Checkbox';
 
 import CloseButton from 'src/shared/components/close-button/CloseButton';
 
-import { TranscriptMetadata } from 'src/shared/types/thoas/metadata';
+import { TranscriptMetadata } from 'src/shared/types/core-api/metadata';
```
Comment (imransl) [inline src/content/app/entity-viewer/gene-view/components/transcripts-filter/TranscriptsFilter.tsx:45]: Another type import

**Comment by imransl** [comment]
Noticed that in some files that type imports were not imported as types. Don't have anything else to comment other than that.

**Comment by azangru** [comment]
I just ran a bulk find-and-replace command last time to update the paths, which is why previously unannotated type imports remained unannotated. I took another pass at the diff, and tried to mark every type import as a type import.

**Review by imransl — APPROVED** [review]
`Strand` is not imported as `type` in some files where it can be, but don't think this should stop us from merging as it is an enum after all :)

**Pull request merged**